nixos/modules/security/pam.nix

```diff
@@ -1520,6 +1520,10 @@ in
 
     environment.etc = mapAttrs' makePAMService config.security.pam.services;
 
+    systemd.packages = optionals config.security.pam.services.login.updateWtmp [ pkgs.pam_lastlog2 ];
+    systemd.services.lastlog2-import.enable = config.security.pam.services.login.updateWtmp;
+    systemd.tmpfiles.packages = optionals config.security.pam.services.login.updateWtmp [ pkgs.pam_lastlog2 ];
```

Comment:
While this "works" since this is the only in nix use of `updateWtmp` (which should probably be updated to be `updateLastlog`, but I didn't go searching for historical reasons), I feel like it should be relying on globbing `config.security.pam.services.*.updateWtmp` - but I'm not sure how to make this happen.